Block/Checkout/Success.php

```diff
@@ -96,10 +104,10 @@ public function isBoletoPayment()
     /**
      * @return null|\string[]
      */
-    public function getBoletoPdfUrl()
+    public function getBoletoData()
     {
         if ($this->isBoletoPayment()) {
-            return $this->getOrder()->getPayment()->getAdditionalInformation('url');
+            return $this->getOrder()->getPayment()->getAdditionalInformation();
```

Comment:
I think we should specify the additional data we want to get instead of everything
```suggestion
            return $this->getOrder()->getPayment()->getAdditionalInformation('action');
```